rust/xaynet-mobile/xaynet_ffi.h

```diff
@@ -117,6 +132,34 @@ enum SETMODEL_DATATYPE {
  */
 #define PARTICIPANT_MADE_PROGRESS (1 << 4)
 
+/**
+ * A new global model is available
+ */
+#define PARTICIPANT_NEW_GLOBAL_MODEL (1 << 5)
+
+/**
+ * The original primitive data type of the numerical values to be masked.
+ */
+enum DataType {
+  /**
+   * Numbers of type f32.
+   */
+  F32 = 0,
```

Comment:
These are constants in C, so I think their names should be a little more specific, like `MODEL_DATATYPE_F32`, etc.

Reply:
Good point I changed it to `MODEL_DATATYPE_F32`